package.json

```diff
@@ -21,33 +21,53 @@
     "jupyterlab-extension"
   ],
   "scripts": {
-    "build": "tsc",
-    "clean": "rimraf lib",
-    "lint": "tslint",
-    "prepare": "npm run build",
-    "watch": "tsc -w"
+    "build": "jlpm run build:lib && jlpm run build:labextension:dev",
+    "build:prod": "jlpm run build:lib && jlpm run build:labextension",
+    "build:labextension": "jupyter labextension build .",
+    "build:labextension:dev": "jupyter labextension build --development True .",
+    "build:lib": "tsc",
+    "clean": "jlpm run clean:lib",
+    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
+    "clean:labextension": "rimraf jupyterlab_vim/labextension",
+    "clean:all": "jlpm run clean:lib && jlpm run clean:labextension",
+    "eslint": "eslint . --ext .ts,.tsx --fix",
```

Comment:
I think the build is failing in the pull request because it does not find `lint`. We need to change `lint` to `eslint` in `.github/workflows/pull-request.yml`.

Reply:
:+1: 

also we need to ensure that jupyterlab 3 is being installed that's what is causing the build to fail